### ISSUE TITLE
Upgrade to dependecychecker 1.3.1

### ISSUE THREAD

**Issue by seanboylan** [dependency-check/DependencyCheck#412]
Hi 

We have been using dependency checker for some time now and I just upgraded to 1.3.1 from a 1.1.1 version. I have a two questions that I am hoping you can help with.

cmdline arguments used:
/opt/app/dependencychecker/bin/dependency-check.sh --app auth --nexus http://mavenrepo.*********.net:8081/nexus/content/groups/releases-only --out /opt/app/auth/ASSE/dependencyreport --format ALL --scan /opt/app/auth

1) Should I remove the --nexus option in favor of the CentralAnalyzer as suggested in the documentation.
Will this increase build times?

2) What is the implication of the 404 from or nexus ? Any way I can determine whats causing this.

[INFO] Analysis Starting
[WARN] Expected 200 result from Nexus, got 404
[WARN] There was an issue getting Nexus status. Disabling analyzer.
[WARN] An error occurred with the .NET AssemblyAnalyzer;
this can be ignored unless you are scanning .NET DLLs. Please see the log for more details.
[ERROR] Exception occurred initializing Assembly Analyzer.
[INFO] Creating the CPE Index
[INFO] CPE Index Created (2019 ms)
[INFO] Analysis Complete (9588 ms)

**Comment by colezlaw** [comment]
The Central analyzer uses the Maven Central repo on the Internet to look up classes. If you have an internal Nexus repository, you can use either one - but if folks are bypassing the internal Nexus repository and getting stuff straight from Central, then the odds are better that it will be found on Central. It may increase build times slightly because it's going to the Internet instead of an internal server to get that information.

The 404 response is because when the Nexus analyzer starts, it attempts to hit the Nexus service to verify that it's indeed a Nexus server running the way it expects it to, but it's not (hence the 404). So as it stands right now, because you've configured Nexus, it disables the Central analyzer, but because the Nexus analyzer gets a 404 initially, it gets disabled as well. The correct URL for a Nexus service probably ends with /service/local - so if your internal Nexus repository is at http://some.host.com/nexus, the service URL is probably http://some.host.com/nexus/service/local

In short, here are the pros and cons of each:
Central:
- GENERALLY more accurate as it's going to have more stuff than your internal repository
- May be slower because it's going to the Internet
- May be missing re-packaged open source libraries that are on your Nexus server
- You may have to configure a proxy for the Central analyzer to work (if you're behind a proxy).

Nexus:
- Will probably only have components that internal folks have requested through that
- May be faster since it's internal
- Probably will not require proxy configuration

**Comment by seanboylan** [comment]
Thank you for the update. Very much appreciated
